docs/root/version_history/v1.18.3.rst

```diff
@@ -0,0 +1,31 @@
+1.18.3 (May 11, 2021)
+======================
+
+Incompatible Behavior Changes
+-----------------------------
+*Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+
+Minor Behavior Changes
+----------------------
+*Changes that may cause incompatibilities for some users, but should not for most*
+
+Bug Fixes
+---------
+*Changes expected to improve the state of the world and are unlikely to have negative effects*
+
+* zipkin: fix timestamp serializaiton in annotations. A prior bug fix exposed an issue with timestamps being serialized as strings.
+
+Removed Config or Runtime
+-------------------------
+*Normally occurs at the end of the* :ref:`deprecation period <deprecated>`
+
+* tls: removed `envoy.reloadable_features.tls_use_io_handle_bio` runtime guard and legacy code path.
```

Comment:
I doubt we removed this in a point release?

/wait

Reply:
@mattklein123 Thnaks for review. i followed below path
https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current
please suggest...

Reply:
You can check the release, but this must be a bug. We would never remove a runtime guard in a point release.

Reply:
Are you saying https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current also not Correct and `envoy.reloadable_features.tls_use_io_handle_bio `should be under "Bug Fixes" Section in version_history file for v1.18.3 ?

Reply:
I'm saying that I don't think this change was in that release, but please check.

Reply:
Ok. Would you please suggest where should i double check as i have only file https://www.envoyproxy.io/docs/envoy/v1.18.3/version_history/current to refer

Reply:
I would look at the branches: https://github.com/envoyproxy/envoy/tree/release/v1.18

Reply:
humm.. Thanks for this .. i checked file https://github.com/envoyproxy/envoy/blob/v1.18.3/docs/root/version_history/current.rst 
and found same entry there as well. please suggest ...